src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.SslProtocols.cs

```diff
@@ -303,7 +303,7 @@ public void DisabledProtocols_SetSslProtocols_ThrowsException(SslProtocols disab
             }
         }
 
-        private static bool SslDefaultsToTls12 => !PlatformDetection.IsWindows7;
```

Comment:
Can you please add reference to the bug # for Fedora27? So that we don't forget to revert it back. Thanks!